e2e-tests/create-cluster_test.go

```diff
@@ -212,7 +329,38 @@ var _ = Describe("CreateCluster", func() {
 
 		By("Waiting on kubevirt machines to be ready")
 		waitForMachineReadiness(2, 0)
+	})
+
+	It("creating a simple externally managed cluster ephemeral VMs", func() {
```

Comment:
I wonder if it won't easier to patch the cluster to add the missing annotation instead of reading it to a yaml file and apply it..

Reply:
> I wonder if it won't easier to patch the cluster to add the missing annotation instead of reading it to a yaml file and apply it..

OK. Got it after second reading.

Reply:
yeah, the issue is that we need the annotation to exist at creation time.  I'm sure there's a more elegant way of modifying yaml, but this was pretty trivial